sdk/logs/src/main/java/io/opentelemetry/sdk/logs/LoggerConfig.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.logs;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.concurrent.Immutable;
+
+@AutoValue
+@Immutable
+public abstract class LoggerConfig {
```

Comment:
Just thinking out loud, not blocking: 

I don't understand how otel enablement combined with specific logging provider enablement would work. We'll have many ways to configure it, which one will be the source of truth?

Reply:
Yeah the pipeline is:
```
Log API (SLF4J) -> Log Framework (Log4j) -> Otel log bridge API -> Otel log SDK
```

- If Log4j disables a logger, the logs will never make it to the otel log bridge API, so any intersecting Otel log SDK config is ignored
- If log4j enables a logger, the Otel log SDK config has an additional opportunity
- If there is overlapping responsibilities, and multiple ways to achieve the same affect, why include Otel log SDK logger config at all?
  - Events are also going to flow through the Otel log SDK, and they will not have any equivalent mechanism.
  - Symmetry with Tracer and Meter is important.

Reply:
Agree, so as a user I'd use log4j config as the first filter that also does perf optimization. Then otel filter is an additional one and you might want to use it to narrow down what goes into otel.

Unrelated to this work, but eventually it'd be nice if OTel config could be used as a source of logging provider configuration.